molgenis-biobank-plots/src/main/frontend/biobank-plots/src/store/index.js

```diff
@@ -0,0 +1,137 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import BootstrapVue from 'bootstrap-vue/dist/bootstrap-vue.esm'
```

Comment:
Goal of M4928 is a demo with working happy flow with minimum development effort. Code review is always good, but not required in this case.